### PR TITLE
HW-2 was done

### PR DESCRIPTION
**Pull request by rudendan**

### CODE REVIEW

File: HW-2/src/main/java/org/example/Main.java [hunk 1]
```diff
@@ -1,7 +1,11 @@
 package org.example;
 
 public class Main {
-    public static void main(String[] args) {
+    public static void main(String[] args)  {
 
+        ValueCalculator calculator = new ValueCalculator(1);
+        calculator.executeWithTwoThreads();
```
Comment (kartius) [inline HW-2/src/main/java/org/example/Main.java]: Please, use one method with input parameter as an amount of threads

**Comment by kartius** [comment]
You don't need to push  .idea and target folders to git

File: HW-2/src/main/java/org/example/Main.java (hunk 1 above)
Comment (rudendan) [reply HW-2/src/main/java/org/example/Main.java]: The method executeWithTwoThreads() was deleted

**Comment by rudendan** [comment]
> You don't need to push .idea and target folders to git
My bad  :( I will be more careful in nexts pull requests

**Pull request merged**